gpkit/_mosek/cli_expopt.py

```diff
@@ -27,13 +28,14 @@ def errorRemoveReadonly(func, path, exc):
         pass
 
 
-def imize_fn(filename):
-    folder = os.path.expanduser("~") + os.sep + "gpkit_tmp"
-    filename = folder + os.sep + filename
+def imize_fn(path=None):
+    if not path:
+        path = tempfile.mkdtemp()
+    filename = path + os.sep + "gpkit_mosek"
     os.environ['PATH'] = (os.environ['PATH'] + ':%s' %
                           settings["mosek_bin_dir"][0])
 
-    def imize(c, A, p_idxs, k):
+    def imize(c, A, p_idxs, k, *args, **kwargs):
```

Comment:
Yup, so that someone could call solve with arguments that a particular solver doesn't use and not have that solver die.